app.js

```diff
@@ -63,7 +63,8 @@ app.use(errorLogger);
 app.use(errors());
 
 
-app.use((err, req, res) => {
+// eslint-disable-next-line no-unused-vars
+app.use((err, req, res, next) => {
   res.status(err.statusCode || 500).send({ message: err.message });
```

Comment:
Можно лучше: унифицировать ответ для ошибок на сервере.
В большинстве случаев клиент не в состоянии поправить ошибку на сервере. Поэтому детали ошибки ему нет смысла отправлять. Обычно детали ошибки логируются на сервере в файл/консоль/менеджер ошибок (типа Sentry, Honeybadger).

Но ответ нужно какой-то отдавать поэтому рекомендую оставить какой-то общий текст. Например: `'Произошла ошибка'`.

```suggestion
  const statusCode = err.statusCode || 500;
  const message = statusCode === 500 ? 'Произошла ошибка' : err.message;
  res.status(statusCode).send({ message });
```

Reply:
Исправил